.github/workflows/ci.yml

```diff
@@ -21,6 +21,27 @@ jobs:
       - run: npm install
       - run: npm run lint:format
       - run: npm run lint
+  jats:
```

Comment:
Nice, would be good to have an equivalent to this on all our jats examples.

Reply:
Yep, let's keep pushing on this here, and can bring it back into myst.